app/code/Magento/Sales/Model/Order/CreditmemoFactory.php

```diff
@@ -147,7 +147,7 @@ protected function canRefundItem($item, $qtys = [], $invoiceQtysRefundLimits = [
         if ($item->isDummy()) {
             if ($item->getHasChildren()) {
                 foreach ($item->getChildrenItems() as $child) {
-                    if (empty($qtys)) {
+                    if (empty($qtys) || (count(array_unique($qtys)) === 1 && end($qtys) == 0)) {
```

Comment:
Any particular reason why you do not want to make === compare?

Reply:
Hi @nuzil 

> Any particular reason why you do not want to make === compare?

When I debug, I saw $qtys will return array value like this:

  array (size=3)
  48 => string '0' (length=1)
  49 => string '0' (length=1)
  50 => string '0' (length=1)

So now I already update code to use === compare

`if (empty($qtys) || (count(array_unique($qtys)) === 1 && (int)end($qtys) === 0)) {`

Please check this,
Thanks